x/stake/cross_stake/pub.go

```diff
@@ -0,0 +1,22 @@
+package cross_stake
```

Comment:
Please also reflect the change in node repo by defining or updating the schemas. Here: `https://github.com/bnb-chain/node/tree/master/app/pub/schemas`

Reply:
Done